slf4j/src/main/scala-2/org/typelevel/log4cats/slf4j/Slf4jLogger.scala

```diff
@@ -18,13 +18,19 @@ package org.typelevel.log4cats.slf4j
 
 import cats.effect.Sync
 import org.typelevel.log4cats.SelfAwareStructuredLogger
-import org.typelevel.log4cats.slf4j.internal._
+import org.typelevel.log4cats.slf4j.internal.{_}
 import org.slf4j.{Logger => JLogger}
+import scala.annotation.nowarn
 
 object Slf4jLogger {
 
-  def getLogger[F[_]: Sync]: SelfAwareStructuredLogger[F] =
-    macro GetLoggerMacros.unsafeCreateImpl[F[_]]
+  @nowarn("cat=unused")
+  private def getLogger[F[_]: Sync]: SelfAwareStructuredLogger[F] = ???
```

Comment:
not sure if this is safe to do, please advice

Reply:
Looking again, I think the solution here is to keep the old macro in place, but make this method `private[slf4j]`

Then it's a source-incompatible change, but in practice calling code won't need editing - the same code will just call the new method